lua/telescope/builtin/files.lua

```diff
@@ -368,10 +368,14 @@ files.current_buffer_fuzzy_find = function(opts)
     })
   end
 
-  local ok, parser = pcall(vim.treesitter.get_parser, bufnr, filetype)
-  if ok then
-    local query = vim.treesitter.get_query(filetype, "highlights")
+  local ts_ok, ts_parsers = pcall(require, 'nvim-treesitter.parsers')
+  if ts_ok then
+    filetype = ts_parsers.ft_to_lang(filetype)
+  end
 
+  local ok, parser = pcall(vim.treesitter.get_parser, bufnr, filetype)
```

Comment:
Sorry, this is small nit, but can you change this to parser_ok? After reading it now, this ok looks lonely 😂

Reply:
Haha, no problem. I get it ;)